src/Interaction.test.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @jest-environment jsdom
+*/
+import Mock from './__mock__/mockFunction.js';
+
+describe('Test for for updating an items completed', () => {
+  test('Testing task status', () => {
+    document.body.innerHTML = '<div class = "todo-container" >'
+    + '<div>'
+      + '<div class = "todo-list">'
+        + '<div id = 1 class = "mydiv">'
+          + '<span class = "desc1">'
+            + '<input type = "checkbox" id = "check1" name = "checking" />'
+            + '<span id = "span21" class = "show" > Task-1</span>'
```

Comment:
- [ ] [Optional][Highly recommended] Its consedered a good practice and good work convention if you include all test files `[...].test.js` and `__mock__` directory inside one folder and you can call it `./tests`. so the file structure will look like that:
```
/src
    - /tests
               - interaaction.test.js
               - __mock__
               - storage.test.js
```

Reply:
### Okay, 🙏🙏🙏 Thank you so much we will do it and it is also better for file management. thank you again for your good comments🙏🙏.